docker/Makefile

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
do we really need a Makefile for this. It would be the first one for this project. If we accept it, we should probably use it also for other purposes.
@stepansnigirev what do you think about Makefiles?

Reply:
I made a note of this in the commit message, but probably should have said something in this PR draft, too:

the makefile is 100% a convenience script, and not required for this setup. 

Having said that, being able to do `make up` after cloning the repo, from any linux machine, and have the entire thing built and running automatically is pretty darn convenient, so I wouldn't necessarily throw it away just because makefiles aren't used elsewhere. It's a nice-to-have.

Also, Makefiles are powerful and very easy to use (just not write...). You should consider using Make, because it has the potential to make building/testing and installing this project a lot easier. There's a reason this ancient technology is sticking around ;) it keeps proving itself. I use it for basically every project out of convenience, even when it's not a C/C++ project. It's a swiss army knife tool for any kind of local build, and a shortcut to a CLI for development purposes (`make {build,run,test,install,rebuild,lint,...}`).

Reply:
Yes, i agree here. It's sometimes annoying though when things get complex which are quite easy in a bas-script are quite complicated in a Makefile.
So i'm happy to keep it. CC @ben-kaufman

Reply:
100%. The trick is realizing when bash script get out of hand, and when makefiles get out of hand...
With makefiles I find the rule of thumb is whenever you find yourself splitting it to multiple files with a whole include hierarchy then it's probably time to reconsider the tooling. It's very powerful and very succinct, so often it goes a long way before it reaches that stage.